geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt

```diff
@@ -28,6 +28,20 @@ Information
    ``service: geoportal`` by ``service: geoportal-advance``, and in service ``alembic`` you should replace
    ``service: alembic`` by ``service: alembic-advance``.
 
+4. Some little changes in the application build:
+   - We no more use custom build commands to build the docker images, we simply use the docker-compose
+     build command, with tow additional environment variable:
+     * ``SIMPLE``: ``TRUE`` or ``FALSE`` if we compile a simple application or not.
+     * ``GIT_HASH``: the git hash of the current version.
+     Then all the build specification will be moved to the ``docker-compose`` files.
```

Comment:
"Then all the build specification will be moved"
->
"The build specification has been moved"

(if I understand correctly)